mobile/lib/widgets/settings/networking_settings/networking_settings.dart

```diff
@@ -102,7 +101,7 @@ class NetworkingSettings extends HookConsumerWidget {
           padding: const EdgeInsets.only(top: 8, left: 16, bottom: 8),
           child: NetworkPreferenceTitle(
             title: "current_server_address".tr().toUpperCase(),
-            icon: currentEndpoint.startsWith('https')
+            icon: currentEndpoint!.startsWith('https')
```

Comment:
You can still access this page before any server URL is specified from the login screen. If there is no `currentEndpoint`, we can just show it as `http` icon instead